src/Module.php

```diff
@@ -1,43 +1,38 @@
 <?php
+
 /**
  * @link      https://github.com/weierophinney/PhlySimplePage for the canonical source repository
- * @copyright Copyright (c) 2012 Matthew Weier O'Phinney (http://mwop.net)
+ * @copyright Copyright (c) 2012-2020 Matthew Weier O'Phinney (https://mwop.net)
  * @license   https://github.com/weierophinney/PhlySimplePage/blog/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace PhlySimplePage;
 
-use Zend\Mvc\Application;
-use Zend\Stdlib\ResponseInterface;
+use Laminas\Mvc\Application;
+use Laminas\Mvc\MvcEvent;
+use Laminas\Stdlib\ResponseInterface;
+
+use function in_array;
+use function str_replace;
 
-/**
- * Module class for use with ZF2
- */
 class Module
 {
     /**
-     * Retrieve application configuration for this module
-     *
-     * @return array
+     * Normalize a cache key
      */
-    public function getConfig()
+    public static function normalizeCacheKey(string $key): string
     {
-        return include __DIR__ . '/config/module.config.php';
+        return str_replace(['/', '\\', '.'], '_', $key);
     }
 
     /**
-     * Provide console usage messages for console endpoints
-     *
-     * @deprecated Since 1.1.0. To be removed in 2.0.0.
-     * @return array
+     * Retrieve application configuration for this module
      */
-    public function getConsoleUsage()
+    public function getConfig(): array
     {
-        return [
-            'phlysimplepage cache clear all' => 'Clear caches for all static pages',
-            'phlysimplepage cache clear --page=' => 'Clear caches for a single static page',
-            ['--page', 'Page name as matched via routing'],
-        ];
+        return include __DIR__ . '/config/module.config.php';
```

Comment:
@weierophinney  `config` directory is one level up. Can you please do the needful. Thanks.

Reply:
@weierophinney  proposed PR https://github.com/phly/PhlySimplePage/pull/20